manifests/0000_50_cluster-monitoring-operator_02-role.yaml

```diff
@@ -374,18 +374,6 @@ rules:
   - /metrics
   verbs:
   - get
-- apiGroups:
```

Comment:
@arajkumar do you have any idea why this is showing up in the diff now, and how it wasn't caught before?

Reply:
Looks like a rebase problem on your side. Can you try once again?

Reply:
This PR is already rebased against latest master. It seems like there is something strange with either the jsonnet code or with how we generate the yaml files. Touching anything in the cluster role in prometheus.jsonnet will also affect the CMO cluster role.

Reply:
So it looks like this change is driven by the deletion of these lines: https://github.com/openshift/cluster-monitoring-operator/pull/1406/files#diff-cf7ce97df96f13b990e1fd5d01c1cb685e705bb4b0fcaf039187a92c029b30acL138-L147

Reply:
Ok, looks like this works as expected. The CMO role is an aggregation of all operand roles: https://github.com/openshift/cluster-monitoring-operator/blob/master/jsonnet/main.jsonnet#L336

Reply:
Thats correct, I was actually looking that specific asset without the actual jsonnet changes :)